src/modules/account/utils/decryptAccount.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line no-unused-vars
+
+// TODO this function should import from SDK
+// eslint-disable-next-line no-unused-vars
+const decryptAES256GCMWithPassword = ({ encryptedPassphrase, password }) => JSON.stringify({
+  privateKey: 'private-key-mock',
+  recoveryPhrase: 'target cancel solution recipe vague faint bomb convince pink vendor fresh patrol',
+});
+
+// eslint-disable-next-line
+export const decryptAccount = (accountSchema, password) => {
+  try {
+    const plainText = decryptAES256GCMWithPassword({
+      encryptedPassphrase: accountSchema,
+      password,
+    });
+    const { privateKey, recoveryPhrase } = JSON.parse(plainText);
+    return { privateKey, recoveryPhrase, error: false };
+  } catch (e) {
+    return { privateKey: null, recoveryPhrase: null, error: true };
```

Comment:
```suggestion
catch(error) {
    return { error };
```

Reply:
@isalga @soroushm believes that the error caught might be unhelpful and that's why the object is returned this way

Reply:
Ok, still I do think that would be better to return the whole error object instead of 'error: true', '{ privateKey: null, recoveryPhrase: null, error }' the error object being defined can work as a condition. Just a comment, decide which implementation you prefer.